### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by erlend-sh** [spicylobstergames/cow-rs#1]
More details to come, but the initial plan is very simple: Do a pretty loose translation to Rust. Not trying too hard to copy the original architecture, but copying the game logic.

Milestone 1: terminal graphics.

Milestone 2: 2D graphics.

**Comment by ricott1** [comment]
Regarding the terminal graphics, do you have already a resolution/frame rate target in mind?

**Comment by erlend-sh** [comment]
Not really; we wanna go as high as possible without adding any complexity.

Might be easiest to scale up 2x or 4x of the established resolution in CoW.
